src/lib/layer-manager.js

```diff
@@ -200,7 +265,51 @@ export default class LayerManager {
     return redraw;
   }
 
+  //
   // PRIVATE METHODS
+  //
+
+  /**
+   * Route click events to layers.
+   * `pickLayer` will call the `onClick` prop of any picked layer,
+   * and `onLayerClick` is called directly from here
+   * with any picking info generated by `pickLayer`.
+   */
+  _onClick(event) {
+    const pos = getPos(event);
+    if (!pos) {
+      return;
+    }
+    const radius = this.pickingRadius;
+    const selectedInfos = this.pickLayer({x: pos.x, y: pos.y, radius, mode: 'click'});
+    if (selectedInfos.length) {
+      const firstInfo = selectedInfos.find(info => info.index >= 0);
```

Comment:
Should we just have a function `pickTopLayer` so we can drop the `find` here? @Pessimistress